project/plugins.sbt

```diff
@@ -13,4 +13,4 @@ lazy val build = (project in file(".")).
 
 (Compile / unmanagedSourceDirectories) += baseDirectory.value.getParentFile / "src" / "main" / "scala"
 
-addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.9.3")
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.11.0")
```

Comment:
I'm in the process of migrating com.github.sbt to the Central Portal. Please update to sbt-ci-release 1.11.0 only after that's done.